scripts/basic_openwrt_config.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo "CONFIG_DEVEL=y"
+if [ "$GLUON_AUTOREMOVE" = "1" ]; then
```

Comment:
As our Makefile sets `GLUON_AUTOREMOVE` to 0 by default, the following might be closer to the `istrue()` check our Lua scripts do:
```suggestion
if [ "$GLUON_AUTOREMOVE" != "0" ]; then
```